content/minfraud/api-documentation/_schemas/ResponseDisposition.mdx

```diff
@@ -76,4 +82,13 @@ import responseJson from '../_examples/response';
       </tbody>
     </table>
   </Property>
+
+  <Property
+    name="rule_label"
+  >
+    The custom rule that was triggered. If you do not have custom rules set up,
+    the triggered custom rule does not have a label, or no custom rule was
+    triggered, the key will not be included in the response.
```

Comment:
This should be: the **field** will not be included in the response.

Reply:
Cool, apologies I probably provided that word.

Reply:
okay, i've fixed those copy errors. apologies!